README.md

```diff
@@ -9,23 +13,29 @@
 <a href="#run-the-app-using-docker">using Docker</a>
 </p>
 
----
- 
-A GTFS Schedule (static) [General Transit Feed Specification (GTFS)](https://gtfs.mobilitydata.org/spec/gtfs-schedule) feed validator, maintained by [MobilityData](www.mobilitydata.org).
 
+<p align="center">
+<a href="https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md">☑️ list of rules implemented</a>
+</p>
 
-<video src="https://user-images.githubusercontent.com/63653518/182135478-f844cbfd-7037-4d02-bd65-f196cabaddeb.mp4" controls="controls" style="max-width: 730px;">
-</video>
+<p align="center">
+<a href="https://github.com/MobilityData/gtfs-validator/blob/master/docs/CONTRIBUTING.md"> 🤝 contribute to the project</a>
+</p>
```

Comment:
nit: my own personal preference here, but the use of lower-case to start each sentence here (contribute to..., list of rules..., using the ...) strikes me as a little weird.  But not a deal breaker.